### PR TITLE
Solution overview - secure systems administration 

### PR DESCRIPTION
**Pull request by oobedan**
Updates Include:
Added some guidance around PAWs and secure administration in the solution overview.

### CODE REVIEW

File: blueprint/overview.md [hunk 1]
```diff
@@ -163,6 +163,22 @@ Collaboration is initially controlled by lists of allowed domains. Individual us
...
+* **Virtual jump host** solution may be built leveraging Azure Virtual Desktop (AVD) session hosts, which includes seamlessly provides Multifactor Authentication (MFA) though Conditional Access policies. This jump host may be the trusted location where admin portals are used, as well as a secure place to administer other hybrid assets. Network security groups can be used to restrict management traffic flows to these hosts only
+* **Restriction of management traffic flows** limited to only critical assets should be implemented where possible. This could be achieved by utilising the Windows Defender native firewall functionality, web filtering, network security groups on cloud hosted jump hosts, and Conditional Access policies.
+
+Review the Australian Government Information Security Manual (ISM) controls for systems management to assist with implementing a secure administration model.
```
Comment (zedzedzed) [inline blueprint/overview.md:180]: Can we link to this? Is it https://www.cyber.gov.au/acsc/view-all-content/publications/secure-administration?

File: blueprint/overview.md [hunk 2]
```diff
@@ -163,6 +163,22 @@ Collaboration is initially controlled by lists of allowed domains. Individual us
...
+* **Web filtering system** to restrict privileged accounts to the set of agency approved admin portals only. This product could be a risk assessed cloud platform or built as an infrastructure as a service server. The web filtering solution should be mandatory on the jump host or PAWs. [Tenant restrictions](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/tenant-restrictions) should be implemented to prevent misuse or data exfiltration to other Microsoft tenancies
+* **Virtual jump host** solution may be built leveraging Azure Virtual Desktop (AVD) session hosts, which includes seamlessly provides Multifactor Authentication (MFA) though Conditional Access policies. This jump host may be the trusted location where admin portals are used, as well as a secure place to administer other hybrid assets. Network security groups can be used to restrict management traffic flows to these hosts only
```
Comment (zedzedzed) [inline blueprint/overview.md:177]: Does the following apply to all forms of secure access or only to jump hosts?

```This jump host may be the trusted location where admin portals are used, as well as a secure place to administer other hybrid assets. Network security groups can be used to restrict management traffic flows to these hosts only```

File: blueprint/overview.md (hunk 1 above)
Comment (oobedan) [reply blueprint/overview.md]: added link

File: blueprint/overview.md (hunk 2 above)
Comment (oobedan) [reply blueprint/overview.md]: applies to these 'jump hosts only'. have updated wording

**Review by zedzedzed — APPROVED** [review]

**Pull request merged**